pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "dbt-coverage"
-version = "0.1.2"
+version = "0.1.3"
```

Comment:
@sweco would it make sense to keep a changelog too?

Reply:
I guess it would make sense to have it. 😊  Do you think I should just create a file or use the [GitHub releases](https://github.com/slidoapp/dbt-coverage/releases) to write the changes?

Reply:
I'd say a standard changelog file (https://keepachangelog.com/en/1.0.0/) would be more than enough

Reply:
Here it is! https://github.com/slidoapp/dbt-coverage/pull/10